android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.java

```diff
@@ -169,6 +171,12 @@ public void onUpdate() {
       }
     }
 
+    // orientation
+    if (getScreenFragment() == null || !getScreenFragment().hasChildScreenWithConfig(getScreen())) {
+      // we check if there is no child that provides config, since then we shouldn't change orientation here (child with index 0 is this header config)
```

Comment:
is this comment outdated? why are we referring to index 0 if no code around here references children nor indices

Reply:
Yes, it is outdated, there was getting child by index before.